Windows/W32Util/Misc.h

```diff
@@ -47,8 +47,8 @@ struct GenericListViewColumn
 struct GenericListViewDef
 {
 	const GenericListViewColumn* columns;
-	int columnCount;
 	int* columnOrder;
+	int columnCount;
```

Comment:
This is another one I'd prefer to revert because it makes the tables that use the struct look less nice.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/19047/commits/87bd591cee2a17d537b75330467cc699318fd3b9